### PR TITLE
fix typo in kubectl run help string

### PR DESCRIPTION
**Pull request by j3ffml**
ref #22849

### CODE REVIEW

**Comment by bgrant0607** [comment]
Please run hack/update-generated-docs.sh

File: docs/user-guide/kubectl/kubectl_run.md [hunk 1]
```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
-<strong>
-The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.2/docs/user-guide/kubectl/kubectl_run.md).
```
Comment (bgrant0607) [inline docs/user-guide/kubectl/kubectl_run.md]: What happened here?

**Comment by bgrant0607** [comment]
Thanks
cc @soltysh

**Comment by jlowdermilk** [comment]
We actually need to fix the update/verify generated docs scripts. We shouldn't be modifying docs in the main repo any more now that they've been moved to the kubernetes.io repo #22863.

**Comment by bgrant0607** [comment]
LGTM

On generated docs: Yes, the doc move left lots of loose ends. That was one.

**Comment by bgrant0607** [comment]
Manual merge @k8s-oncall

**Comment by eparis** [comment]
This PR is included in #22874 which is slated to be included in the release-1.2 branch.
Please verify that the cherry-pick in that PR looks correct.

**Pull request merged**